dart/lib/src/builder.dart

```diff
@@ -558,7 +555,7 @@ class _StackValue {
     for(var i = 0; i < 4; i++) {
       final width = 1 << i;
       final offsetLoc = size + BitWidthUtil.paddingSize(size, width) + index * width;
-      final offset = offsetLoc - _offset;
+      final offset = offsetLoc - _offset!;
```

Comment:
consider adding an `assert` outside the for loop `assert(_offset != null)`

Reply:
done